wordle-web/components/settings-dialog.vue

```diff
@@ -37,7 +42,13 @@
               <v-list>
                 <v-list-item-group>
                   <v-list-item @click="purpleTheme">
-                    <v-list-item-title> Purple </v-list-item-title>
+                    <v-list-item-title> Lavender </v-list-item-title>
+                  </v-list-item>
+                  <v-list-item @click="greenTheme">
+                    <v-list-item-title> Forest </v-list-item-title>
+                  </v-list-item>
+                  <v-list-item @click="blueTheme">
+                    <v-list-item-title> Blueberry </v-list-item-title>
```

Comment:
It would be nice to have a button to reset the color theme to default, but that wasn't a homework requirment. Good job here!